portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +32,17 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> messages = new ArrayList();
+  static final String taskName = "Comments";
```

Comment:
this should be TASK_NAME. also can it be private?

style guide: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names